auth/src/main/java/com/firebase/ui/auth/provider/GoogleProvider.java

```diff
@@ -62,8 +65,7 @@ public GoogleProvider(FragmentActivity activity, IDPProviderParcel parcel, @Null
                 .requestIdToken(mClientId);
 
         // Add additional scopes
-        String[] extraScopes = mActivity.getResources().getStringArray(R.array.google_permissions);
```

Comment:
Bonus points: since we're removing these scopes as resources, maybe we could log a big warning message if we see them defined and they're non empty.  Something like "DEVELOPER_WARNING: you have defined R.array.google_permissions but that is no longer respected as of FirebaseUI 1.0.0.  See README"